src/models/Settings.js

```diff
@@ -138,7 +138,9 @@ function (Okta, Q, Errors, BrowserFeatures, Util, Logger, OAuth2Util, config) {
       'customButtons': ['array', false, []],
 
       //Registration
-      'registration.click': 'function'
+      'registration.click': 'function',
+      'registration.clientId': 'string'
```

Comment:
There is a JIRA to use clientId property we already have https://oktainc.atlassian.net/browse/OKTA-135949